source/streaming/tests/streaming_utils/testdata/vr200_invalid_time_series.template.json

```
@@ -1,49 +1,41 @@
 {
-  "mRID": "tId2020-12-01T13:16:29.330Z",
-  "MessageReference": "mId2020-12-01T13:16:29.330Z",
-  "MarketDocument": {
-    "mRID": "hId2020-12-01T13:16:29.330Z",
-    "Type": "E66",
-    "CreatedDateTime": "2020-12-01T13:16:29.33Z",
-    "SenderMarketParticipant": {
-      "mRID": "8100000000030",
-      "qualifier": "VA",
-      "name": null,
-      "Type": "MDR"
+  "document": {
+    "id": "hId2020-12-01T13:16:29.330Z",
+    "requestDateTime": "2020-12-15T13:15:11.8349163Z",
+    "type": 1,
+    "createdDateTime": "2020-12-01T13:16:29.33Z",
+    "sender": {
+      "id": "8100000000030",
+      "businessProcessRole": 4
```

nit, use the correct recipient role:
```suggestion
      "businessProcessRole": 6
```